SFDigitalServices/sf-dahlia-web
# DAH-1170 update translation and stylings for lottery modal

[DAH-1170](https://sfgovdt.jira.com/jira/software/c/projects/DAH/boards/122?modal=detail&selectedIssue=DAH-1170)

- bump version of ui-components to include ExpandableContent className prop
- update translation string to mirror Angular implementation
- add padding around ExpandableContent button to mirror Angular implementation
- refactor ListingDetailsLotteryRanking to allow for stylings to mirror Angular Implementation
- refactor ListingDetailsLotteryRanking to use same implementation of lottery bucket details

This ticket addresses an [erroneous translation within the Lottery Rankings Modal](https://user-images.githubusercontent.com/5691923/173703877-ec161a6c-73a2-4210-bcd7-274cfd43e131.png) and making the styles of the Lottery Modal to be more in-line with Angular. While improving the styling inside of the ListingDetailsLotteryPreferences, it was noticed the lottery bucket details were implemented differently despite them being the same container, so those were consolidated and that fixed an additional styling discrepancy.

**Review Instructions**
- Ensure that the Expanded Button text and styling is the same in the react version as the angular version ![image](https://user-images.githubusercontent.com/5691923/173936556-3d985f42-6959-451d-9ec2-ecd91c8de6db.png)
- Check the borders go to the full width of the lottery results modal beneath the "GENERAL POOL" section and "LIVE OR WORK IN SAN FRANCISCO PREFERENCE"
![image](https://user-images.githubusercontent.com/5691923/173936014-8963df4f-73d2-49aa-ad5e-a1202f3f7fe3.png)
- Ensure the General Pool section padding/styling is consistent with other preference sections (e.g. CERTIFICATE OF PREFERENCE (COP)
